code/drasil-docLang/Drasil/DocumentLanguage/TraceabilityGraph.hs

```diff
@@ -49,7 +49,7 @@ mkGraphNodes entries si = (traceMReferees entryF cdb)
 -- Testing new graphnodes function
 -- | Gets the nodes of a graph based on a list of sections we want to examine and the system information.
 mkGraphNodes :: TraceViewCat -> SystemInformation -> ([UID], [String])
-mkGraphNodes entry si = (nodeContents, map checkUID' nodeContents)
+mkGraphNodes entry si = (nodeContents, map (\x -> checkUID' x si) nodeContents)
```

Comment:
This is an indication that `checkUID'` should probably take its arguments in the opposite order.